### PR TITLE
Fixes BZ 2088565. Extra information on how to use the RHCS python external script with a Metro DR config

### PR DESCRIPTION
**Pull request by likid0**

### CODE REVIEW

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc [hunk 1]
```diff
@@ -120,7 +120,84 @@ After the ODF operators are installed, select *Create StorageSystem* and choose
...
+----
+ceph3.example.com has address 10.0.40.24
+ceph6.example.com has address 10.0.40.66
+----
+
+Execute the ceph-external-cluster-details-exporter.py with the parameters configured for our first ocp managed cluster `cluster1`.
+
+[source,role="execute"]
+----
+python3 ceph-external-cluster-details-exporter.py --rbd-data-pool-name rbdpool --rgw-endpoint 10.0.40.24:8080 --run-as-user client.odf.cluster1 > ocp-cluster1.json
```
Comment (netzzer) [inline training/modules/ocs4/pages/odf4-metro-ramen.adoc:183]: Add note or additional text that makes it clear this is an example for IP address from above Example output.

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc [hunk 2]
```diff
@@ -120,7 +120,84 @@ After the ODF operators are installed, select *Create StorageSystem* and choose
...
+----
+
+Execute the ceph-external-cluster-details-exporter.py with the parameters configured for our first ocp managed cluster `cluster2`
+
+[source,role="execute"]
+----
+python3 ceph-external-cluster-details-exporter.py --rbd-data-pool-name rbdpool --rgw-endpoint 10.0.40.66:8080 --run-as-user client.odf.cluster2 > ocp-cluster2.json
```
Comment (netzzer) [inline training/modules/ocs4/pages/odf4-metro-ramen.adoc:190]: Add note or additional text that makes it clear this is an example for IP address from above Example output.

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc [hunk 3]
```diff
@@ -120,7 +120,84 @@ After the ODF operators are installed, select *Create StorageSystem* and choose
...
+----
+
+
+
+Save the two files generated in the bootstrap cluster(ceph1) ocp-cluster1.json and ocp-cluster2.json to your
+local machine.
+* Use the contents of file ocp-cluster1.json on the OCP console on cluster1 where external ODF is being deployed.
```
Comment (netzzer) [inline training/modules/ocs4/pages/odf4-metro-ramen.adoc:197]: This is not rendering correctly.

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc [hunk 4]
```diff
@@ -120,7 +120,84 @@ After the ODF operators are installed, select *Create StorageSystem* and choose
...
+----
+python3 ceph-external-cluster-details-exporter.py --rbd-data-pool-name rbdpool --rgw-endpoint 10.0.40.66:8080 --run-as-user client.odf.cluster2 > ocp-cluster2.json
+----
+
+
+
+Save the two files generated in the bootstrap cluster(ceph1) ocp-cluster1.json and ocp-cluster2.json to your
+local machine.
+* Use the contents of file ocp-cluster1.json on the OCP console on cluster1 where external ODF is being deployed. 
+* Use the contents of file ocp-cluster2.json on the OCP console on cluster2 where external ODF is being deployed.
```
Comment (netzzer) [inline training/modules/ocs4/pages/odf4-metro-ramen.adoc:198]: This is not rendering correctly.

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc (hunk 1 above)
Comment (likid0) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:183]: Done.

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc [hunk 5]
```diff
@@ -120,7 +120,84 @@ After the ODF operators are installed, select *Create StorageSystem* and choose
...
+----
+
+To know more about the External ODF deployment options, see
+https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/deploying_openshift_data_foundation_in_external_mode/index#overview-of-deploying-in-external-mode_rhodf[ODF external mode deployment.]
+
+At a minimum, we need to use the following three flags with the
+ceph-external-cluster-details-exporter.py script:
+
+* **--rbd-data-pool-name** : With the name of the RBD pool we created during RHCS deployment for OCP, in  our example, the pool is called `rbdpool`.
+* **--rgw-endpoint** : With the RGW IP of the RGW daemon running on the same site as the OCP cluster, we are configuring.
```
[the remark this answers is not in the document]
Comment (likid0) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:153]: I would say that for metroDR is a safe bet to make it a requirement as we are also configuring RGW in the RHCS section, and also Noobaa is needed for the MetroDR config, so it makes sense to use the S3 RGW from the external cluster as a backingstore for Noobaa(it's the default with on-premise deployments).   I have tried to specify the values that change on the ODF install wizard per cluster

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc (hunk 3 above)
Comment (netzzer) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:207]: Fixed

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc (hunk 4 above)
Comment (netzzer) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:208]: Fixed

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc (hunk 1 above)
Comment (netzzer) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:188]: Need to add `.` at end of new NOTE.

Comment (netzzer) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:188]: Fixed

File: training/modules/ocs4/pages/odf4-metro-ramen.adoc (hunk 2 above)
Comment (netzzer) [reply training/modules/ocs4/pages/odf4-metro-ramen.adoc:199]: Fixed

**Review by netzzer — APPROVED** [review]

**Pull request merged**